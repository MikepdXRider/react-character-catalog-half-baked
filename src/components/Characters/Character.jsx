export default function Character({ character }) {
  return (
    <figure>
      <img src={character.image} alt={character.name} />
      <figcaption>
        <h3>{character.name}</h3>
        <p>{character.species}</p>
        <p>Status: {character.status}</p>
      </figcaption>
    </figure>
  );
}
