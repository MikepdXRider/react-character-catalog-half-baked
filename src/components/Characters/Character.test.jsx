import {render, screen} from '@testing-library/react'
import Character from './Character.jsx'


it('character details snapshot', async () => {
    const { container } = render(
        <Character 
        character={{
            image: 'test-image',
            name: 'test-name',
            species: 'test-species',
            status: 'test-status'
        }}
        />
    );

    expect(container).toMatchSnapshot();
})