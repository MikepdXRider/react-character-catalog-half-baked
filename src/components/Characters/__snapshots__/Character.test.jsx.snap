// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`character details snapshot 1`] = `
<div>
  <figure>
    <img
      alt="test-name"
      src="test-image"
    />
    <figcaption>
      <h3>
        test-name
      </h3>
      <p>
        test-species
      </p>
      <p>
        Status: 
        test-status
      </p>
    </figcaption>
  </figure>
</div>
`;
